designer/webpack.config.cjs

```diff
@@ -75,8 +81,8 @@ module.exports = {
             options: {
               sassOptions: {
                 includePaths: [
-                  join(__dirname, '../node_modules'),
-                  join(__dirname, 'node_modules')
+                  join(__dirname, 'node_modules'),
+                  join(__dirname, '../node_modules')
```

Comment:
This has been swapped around to put the project **node_modules** last

We want to include GOV.UK Frontend v4.8.0 from **designer/node_modules** first